.eslintrc-markdown

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "./.eslintrc",
+  "root": true,
+  "plugins": [
+    "markdown"
+  ],
+  "rules": {
+    "class-methods-use-this": 0,
+    "import/no-unresolved": 0,
+    "import/no-extraneous-dependencies": 0,
+    "import/extensions": 0,
+    "max-len": 0,
```

Comment:
I don't really like this rule in general; it's basically a poor proxy for limiting complexity. All the lines it warned on weren't worth arbitrarily splitting up.